src/routes.json

```diff
@@ -7,6 +7,12 @@
         "title": "Imagine, Program, Share"
     },
     {
+        "name": "splash-redirect",
+        "pattern": "^//?$",
```

Comment:
Actually this is gonna cause an infinite redirect. It should be `^///?$` in that case.